source/Generic/PlayNotes/PlayNotesSettings.cs

```diff
@@ -84,6 +84,8 @@ public PlayNotesSettingsViewModel(PlayNotes plugin)
                 maxWidthBinding.RelativeSource = relativeSource;
                 imageStyle.Setters.Add(new Setter(Image.MaxWidthProperty, maxWidthBinding));
             }
+            markdownStyle.Setters.Add(new Setter(FlowDocument.FontFamilyProperty, ResourceProvider.GetResource("FontFamily") as FontFamily));
+            markdownStyle.Setters.Add(new Setter(FlowDocument.FontSizeProperty, ResourceProvider.GetResource("FontSize")));
```

Comment:
Have you checked if both resources are available in Fullscreen Mode too? I haven't touched Fullscreen mode development so I'm not sure if it uses the same resources or not.

Reply:
It seems it doesn't: https://github.com/JosefNemec/Playnite/blob/4c03306ff4bf62ed2a244b20543f2839eb5483cf/source/Playnite.FullscreenApp/Themes/Fullscreen/Default/Constants.xaml#L37-L40

Apparently it uses `FontTitilliumWebLight` but perhaps it would make sense to get the `TextBlockBaseStyle` and grab its `FontFamily`, just in case it was changed in the theme.

Reply:
You can use `ApplicationInfo.Mode` in `plugin.PlayniteApi` to obtain the Playnite mode.

Reply:
Is there a fullscreen theme which supports Play Notes plugin? I've made some changes, and I'd like to test them.

Reply:
I've tried to test the plugin by injecting it in default fullscreen Playnite theme. It looks that it has multiple problems.

![](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/2903496/8f0d4dfc-da85-4dbe-9f94-56556aafb3a1)

1. All icons are missing.
2. Base control is unstyled, which forces background to be white. I've tried to inject TabControl.xaml from default theme, but it didn't work.

But if you'll select the font, it looks like it's correct:
![](https://github.com/darklinkpower/PlayniteExtensionsCollection/assets/2903496/473f2fea-dc9f-4c5e-9f24-1fffd42675be)

The font is bigger than description though, but the problem is that in default theme the description size is hard-coded:

https://github.com/JosefNemec/Playnite/blob/4c03306ff4bf62ed2a244b20543f2839eb5483cf/source/Playnite.FullscreenApp/Themes/Fullscreen/Default/Views/GameDetails.xaml#L118-L122

Even though the new code works, more work is needed to be actually usable in fullscreen mode.

Reply:
The reason is that FS mode doesn't have styling for TabControl by default but this change was required to discard the possibility of crashing on startup